src/ui/units/connections/components/WrappedLoader/WrappedLoader.scss

```diff
@@ -4,9 +4,17 @@
     display: flex;
     align-items: center;
     justify-content: center;
-    min-height: calc(100vh - #{$headerHeight});
+    position: relative;
+    height: 100vh;
 
-    &_with-aside-header {
-        min-height: calc(100vh - #{$actionPanelHeight});
+    &_with-height-offset {
+        height: calc(100vh - #{$headerHeight});
+        top: -#{$headerHeight / 2};
+    }
+
+    &_with-height-offset#{&}_with-aside-header {
+        height: calc(100vh - #{$actionPanelHeight});
+        // A slight asymmetry helps to remove the loader "jump" when the actions panel appears
+        top: -#{$actionPanelHeight / 2};
```

Comment:
<details>
<summary>This styles help to avoid such behaviour:</summary>

![conn_loader_bug_GIF](https://github.com/datalens-tech/datalens-ui/assets/26501073/c3f16175-0396-4e58-8aba-84bb1ad37b2b)


<\details>